l10n_it_appointment_code/data/appointment_code_data.xml

```diff
@@ -0,0 +1,95 @@
+<odoo>
+  <data noupdate="1">
+    <record model="appointment.code" id="1">
+      <field name="code">1</field>
+      <field name="name">Rappresentante legale, negoziale o di fatto, socio
+amministratore
+      </field>
+    </record>
+    <record model="appointment.code" id="2">
+      <field name="code">2</field>
+      <field name="name">Rappresentante di minore, inabilitato o interdetto,
+amministratore di sostegno, ovvero curatore dell’eredità giacente,
+amministratore di eredità devoluta sotto condizione sospensiva o in
+favore di nascituro non ancora concepito
+      </field>
+    </record>
+    <record model="appointment.code" id="3">
+      <field name="code">3</field>
+      <field name="name">Curatore fallimentare</field>
+    </record>
+    <record model="appointment.code" id="4">
+      <field name="code">4</field>
+      <field name="name">Commissario liquidatore (liquidazione coatta
+amministrativa ovvero amministrazione straordinaria)
+      </field>
+    </record>
+    <record model="appointment.code" id="5">
+      <field name="code">5</field>
+      <field name="name">Custode giudiziario (custodia giudiziaria), ovvero
+amministratore giudiziario in qualità di rappresentante dei beni
+sequestrati ovvero commissario giudiziale (amministrazione
+controllata)
+      </field>
+    </record>
+    <record model="appointment.code" id="6">
+      <field name="code">6</field>
+      <field name="name">Rappresentante fiscale di soggetto non residente
+      </field>
+    </record>
+    <record model="appointment.code" id="7">
+      <field name="code">7</field>
+      <field name="name">Erede</field>
+    </record>
+    <record model="appointment.code" id="8">
+      <field name="code">8</field>
+      <field name="name">Liquidatore (liquidazione volontaria)</field>
+    </record>
+    <record model="appointment.code" id="9">
+      <field name="code">9</field>
+      <field name="name">Soggetto tenuto a presentare la dichiarazione ai fini
+IVA per conto del soggetto estinto a seguito di operazioni
+straordinarie o altre trasformazioni sostanziali soggettive
+(cessionario d’azienda, società beneficiaria, incorporante,
+conferitaria, ecc.); ovvero, ai fini delle imposte sui redditi,
+rappresentante della società beneficiaria (scissione) o della società
+risultante dalla fusione o incorporazione
```

Comment:
Dovrebbe essere aggiunto uno spazio per ogni a capo presente, in questo e negli altri codici. 
Altrimenti succede questo:

![Peek 05-05-2021 11-38](https://user-images.githubusercontent.com/227940/117123364-35403780-ad97-11eb-8edf-8f4139d12182.gif)

Inoltre, dove sono presenti degli a capo vengono aggiunti molti spazi al termine della stringa.
Es.:

![Peek 05-05-2021 11-43](https://user-images.githubusercontent.com/227940/117123390-3e310900-ad97-11eb-9842-a7910f0a1d15.gif)